profiles/base/package.use.mask

```diff
@@ -6,6 +6,10 @@
 
 # New entries go on top.
 
+# Alfred Wingate <parona@protonmail.com> (2025-05-19)
+# Upstream only supports amd64 and arm64. Other targets will fail to build.
```

Comment:
What fails to build, just the tests? (maybe clarify that if so, that it masks test kind of imply it but it sounded confusing for me).

If it actually fails to build one some arches and/or multilib then we may need to permanently drop keywords instead (could argue that we should either way if can't even run tests and mask USE=svt-av1 on these arches, maybe also drop multilib and make ffmpeg only use it with native abi like rav1e).

Reply:
In this case upstream has target specific ifdefs for some symbols, if you enable tests it will fail to build tests on the other targets.

I'm also attracted to the idea of removing multilib from this package, removing all non supported keywords may be a step too far though even if upstream explicitly does not support this ([ESPECIALLY 32-bit targets](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/5def505f7f193d890be61e869831378f212a07bd/CMakeLists.txt#L44))